### DISCUSSION TITLE
Output Coding Data Frame berubah bentuk tampilan

### DISCUSSION THREAD

**Discussion by kenanga727** [teamalgoritma/community#853] [Q&A] [answered]
<img width="633" alt="Pertanyaan_Rafi_Yoda Night" src="https://github.com/teamalgoritma/community/assets/118080291/cd99235a-2e77-4de9-a3d4-71fdfcd8bdfe">

**Comment by Yusraf002** [comment, marked as answer]
Dear @kenanga727 

Mengenai kendala yang Anda alami disini terjadi karena ketika melakukan replacement pada kolom return disimpan langsung kedalam data frame clients sehingga data frame clients tertimpa dengan hasil pengubahan kolom return.

seperti pada contoh dipanduan sebaiknya hasil pengubahan disimpan dalam objek kolom yang sama, dengan pendekatan seperti berikut.
`clients['Consulting Sales'] = clients['Consulting Sales'].str.replace('IDR','')`

Untuk mengatasi hal tersebut, Anda dapat mengubah assignment objek sesuai dengan nama kolom yang ingin Anda ubah, seperti yang disampaikan berikut ini:
`clients['Returns'] = kode Anda`
Kemudian lakukan run all chunk above.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

Best Regards,

Team Algoritma
